src/gasman.c

```diff
@@ -1883,88 +1875,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
To resurrect this discussion: Well, it seems that at least @rbehrends thinks that this code could/should be resurrected, as he did so in his PR #57. But of course it could also be re-added later on... but given that this is code we might want to use again "soon", why not just leave it for now? It's not as if it was actively hurting us... :)